.github/workflows/kind.yml

```diff
@@ -44,22 +47,11 @@ jobs:
         # `ref` is inferred as the new commit in the "caller" repository
 
     - name: Checkout Policy Framework
-      # Checkout a stable branch when called from another repository
-      if: ${{ github.event.repository.name != 'governance-policy-framework' }}
       uses: actions/checkout@v2
       with:
         repository: stolostron/governance-policy-framework
         path: framework
-        ref: ${{ github.event.pull_request.base.ref }} # like main or release-*
-
-    - name: Checkout Policy Framework
-      # Checkout like "usual" when called from this repository
-      if: ${{ github.event.repository.name == 'governance-policy-framework' }}
-      uses: actions/checkout@v2
-      with:
-        # `repository` is inferred as the "caller" repository 
-        path: framework
-        # `ref` is inferred as the new commit
+        ref: ${{ env.RELEASE_BRANCH }} # like main or release-*
```

Comment:
Let's say I have a PR in this repo that changes some tests. I want this workflow to test my changes... does it do that? It looks like it would just run the old tests - am I missing something?

Reply:
Ah, I see what you mean. That makes sense--I'll revert it.